Classes/PKLog.swift

```diff
@@ -38,5 +38,10 @@ import Log
      PKLog.warning(one, two, three, separator: " - ")
      PKLog.error(error, terminator: "\n")
  */
-public let PKLog = Logger()
+
+public let PKLog: Logger = {
+    let logger = Logger()
+    logger.minLevel = .trace
```

Comment:
Let's start using DEV/PROD convention. when we compile and release need to set it to production level, otherwise use dev.
For dev use debug as default.
@noamtamim what do you say - for production warning or info?